libplatsupport/mach_include/spacemit/platsupport/mach/timer.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2025, 10xEngineers 
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
```

Comment:
Please add a comment here that explains which timers you are using. This is especially important because the timers have vague names (TIMER1 etc.), and each timer peripheral consists of three timers/counters.

Normally this would also be the place to mention the DTS compat string, but that's unknown for this timer. Its probably an existing timer peripheral used before in other SoCs, we just don't know which one.